extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -346,538 +355,579 @@ A **subtree file** is a binary file that contains availability information for a
+
+* **Type**: `availability`
+* **Required**:  &#10003; Yes
+
+#### Subtree.contentAvailability
+
+The availability of the content of the tiles of the subtree. The structure of the content availability bitstream matches the tile availability bitstream. Content availability is determined by a single bit, 1 meaning content exists at that spatial index, and 0 meaning it does not. If content availability is 1 its corresponding tile availability must also be 1. If content availability is 0 and its corresponding tile availability is 1 then the tile is considered to be an empty tile. If `tile.content` is defined, then `contentAvailability` is required. Otherwise, `contentAvailability is disallowed.
+
+* **Type**: `availability`
+* **Required**: No
+
+#### Subtree.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+#### Subtree.extras
+
+* **Type**: `any`
+* **Required**: No
+
+## Appendix A: Availability Indexing
+
+### Converting from Tile Coordinates to Morton Index
+
+A [Morton index](https://en.wikipedia.org/wiki/Z-order_curve) is computed by interleaving the bits of the `(x, y)` or `(x, y, z)` coordinates of a tile. Specifically:
+
+```
+quadtreeMortonIndex = interleaveBits(y, x)
+octreeMortonIndex = interleaveBits(z, y, x)
 ```
 
-Subtree JSON chunk in `subtrees/6/0/0/0.subtree`:
+For example:
 
-```json
-{
-  "buffers": [
-    {
-      "byteLength": 84
-    }
-  ],
-  "bufferViews": [
-    {
-      "buffer": 0,
-      "byteOffset": 0,
-      "byteLength": 10
-    },
-    {
-      "buffer": 0,
-      "byteOffset": 10,
-      "byteLength": 10
-    },
-    {
-      "buffer": 0,
-      "byteOffset": 10,
-      "byteLength": 64
-    }
-  ],
-  "tileAvailability": {
-    "bufferView": 0
-  },
-  "contentAvailability": {
-    "bufferView": 1
-  },
-  "childSubtreeAvailability": {
-    "bufferView": 2
-  }
-}
+```
+// Quadtree
+interleaveBits(0b11, 0b00) = 0b1010
+interleaveBits(0b1010, 0b0011) = 0b10001101
+interleaveBits(0b0110, 0b0101) = 0b00111001
+
+// Octree
+interleaveBits(0b001, 0b010, 0b100) = 0b001010100
+interleaveBits(0b111, 0b000, 0b111) = 0b101101101
 ```
 
-File structure:
+![Morton Order](figures/morton-indexing.png)
+
+### Availability Bitstream Lengths
+
+| Availability Type | Length (bits) | Description |
+|-------------------|---------------|-------------|
+| Tile availability | `(N^subtreeLevels - 1)/(N - 1)` | Total number of nodes in the subtree |
+| Content availability | `(N^subtreeLevels - 1)/(N - 1)` | Since there is at most 1 content per tile, this is the same length as tile availability |
```

Comment:
```suggestion
| Content availability | `(N^subtreeLevels - 1)/(N - 1)` | Since there is at most one content per tile, this is the same length as tile availability |
```